python/cudf_polars/cudf_polars/dsl/translate.py

```diff
@@ -215,16 +215,16 @@ def _(node: pl_ir.Scan, translator: Translator, schema: Schema) -> ir.IR:
         reader_options, cloud_options = map(json.loads, options)
     file_options = node.file_options
     with_columns = file_options.with_columns
-    n_rows = file_options.n_rows
-    if n_rows is None:
-        n_rows = -1  # All rows
-        skip_rows = 0  # Don't skip
-    else:
-        # TODO: with versioning, rename on the rust side
-        skip_rows, n_rows = n_rows
-
     row_index = file_options.row_index
     include_file_paths = file_options.include_file_paths
+
+    pre_slice = file_options.n_rows
```

Comment:
Curious, was this just a stylistic move?

Reply:
I wanted to match name on the rust side which is `pre_slice`